libres/lib/include/ert/analysis/update.hpp

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
what is being suppressed?

Reply:
Repeated includes into the same translation unit.